cwl/src/test/scala/wdl4s/cwl/CwlWorkflowWomSpec.scala

```diff
@@ -109,7 +109,10 @@ class CwlWorkflowWomSpec extends FlatSpec with Matchers {
 
     val nodes = wfd.innerGraph.nodes
 
-    nodes collect { case gin: GraphInputNode => gin.name } should be(Set("file:///Users/danb/wdl4s/r.cwl#pattern"))
+    val graphInputNodes = nodes collect { case gin: GraphInputNode => gin }
+    graphInputNodes should have size 1
+    val patternInputNode = graphInputNodes.head
+    patternInputNode.name should be("file:///Users/danb/wdl4s/r.cwl#pattern")
```

Comment:
Because there's an equality check using `patternInputNode` at the end so I needed access to it.
I don't see a reason for graphInputNodes though I can inline that.

Reply:
Actually `graphInputNodes` is used both to check its size and get its head

Reply:
This is all ToL but: yes, on the unhappy path. And if the sets mismatched it should be clear why. Anyway, doesn't matter.